src/uu/lsmem/src/lsmem.rs

```diff
@@ -548,4 +629,35 @@ pub fn uu_app() -> Command {
                 .help("print SIZE in bytes rather than in human readable format")
                 .action(ArgAction::SetTrue),
         )
+        .arg(
+            Arg::new(options::NOHEADINGS)
+                .short('n')
+                .long("noheadings")
+                .help("don't print headings")
+                .action(ArgAction::SetTrue),
+        )
+        .arg(
+            Arg::new(options::JSON)
+                .short('J')
+                .long("json")
+                .help("use JSON output format")
+                .action(ArgAction::SetTrue)
+                .conflicts_with_all(&[options::PAIRS, options::RAW]),
+        )
+        .arg(
+            Arg::new(options::PAIRS)
+                .short('P')
+                .long("pairs")
+                .help("use key=\"value\" output format")
+                .action(ArgAction::SetTrue)
+                .conflicts_with_all(&[options::JSON, options::RAW]),
+        )
+        .arg(
+            Arg::new(options::RAW)
+                .short('r')
+                .long("raw")
+                .help("use raw output format")
+                .action(ArgAction::SetTrue)
+                .conflicts_with_all(&[options::JSON, options::PAIRS]),
```

Comment:
clippy complains about some needless borrows:
```suggestion
                .conflicts_with_all([options::PAIRS, options::RAW]),
        )
        .arg(
            Arg::new(options::PAIRS)
                .short('P')
                .long("pairs")
                .help("use key=\"value\" output format")
                .action(ArgAction::SetTrue)
                .conflicts_with_all([options::JSON, options::RAW]),
        )
        .arg(
            Arg::new(options::RAW)
                .short('r')
                .long("raw")
                .help("use raw output format")
                .action(ArgAction::SetTrue)
                .conflicts_with_all([options::JSON, options::PAIRS]),
```